emails/2023-08-11.md

```diff
@@ -0,0 +1,25 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+Last week when we were trying to come up for a name for the [React + GraphQL + Dolt stack](https://www.dolthub.com/blog/2023-08-02-workbench-architecture-and-rgd-stack/) and coined "the rugged stack", I joked that people that work with Dolt are rugged because they do a lot of yard work, branches and logs and stuff. Better than GPT?
```

Comment:
I feel like seeing the acronym makes it clearer why it's called rugged
```suggestion
Last week when we were trying to come up for a name for the [React + GraphQL + Dolt (RGD) stack](https://www.dolthub.com/blog/2023-08-02-workbench-architecture-and-rgd-stack/) and coined "the rugged stack", I joked that people that work with Dolt are rugged because they do a lot of yard work, branches and logs and stuff. Better than GPT?
```